tests/nlu/base/test_persistor.py

```diff
@@ -79,7 +79,7 @@ def test_list_models_method_in_GCSPersistor():
     # noinspection PyUnusedLocal
     def mocked_init(self, *args, **kwargs):
         self._model_dir_and_model_from_filename = lambda x: {
-            "blob_name": ("project", "model_name")
+            "blob_name": ("project", "model_class")
```

Comment:
I guess that was not intended.

Reply:
hm, apparently rename function in pycharm is far from perfect, thanks for catching